### PR TITLE
Document the MaxUnavailableStatefulSet feature

### PR DESCRIPTION
**Pull request by tengqm**
KEP: kubernetes/enhancements#1010
Upstream: kubernetes/kubernetes#82162

### CODE REVIEW

**Comment by mehabhalodiya** [comment]
enhancement ref: https://github.com/kubernetes/enhancements/issues/961
/assign

**Comment by krmayankk** [comment]
i think there is more to document here for this feature

**Comment by tengqm** [comment]
> i think there is more to document here for this feature

@krmayankk It would be nice if you or someone can help provide more docs on this.

File: content/en/docs/concepts/workloads/controllers/statefulset.md
Comment (sftim) [inline content/en/docs/concepts/workloads/controllers/statefulset.md:302]: I'd write:
```suggestion
that are running with the the `MaxUnavailableStatefulSet`
[feature gate](/docs/reference/commmand-line-tools-reference/feature-gates/)
enabled.
```

**Review by sftim — COMMENTED** [review]
Markdown etc LGTM

**Comment by krmayankk** [comment]
> > i think there is more to document here for this feature
> 
> @krmayankk It would be nice if you or someone can help provide more docs on this.

i am writing a blog on that, we can use that as reference

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md
Comment (divya-mohan0209) [inline content/en/docs/reference/command-line-tools-reference/feature-gates.md:940]: Nit.
```suggestion
  of a StatefulSet. The field specifies the maximum number of Pods
```

**Review by divya-mohan0209 — DISMISSED** [review]
Grammatical nit, otherwise LGTM.

File: content/en/docs/concepts/workloads/controllers/statefulset.md
Comment (sftim) [inline content/en/docs/concepts/workloads/controllers/statefulset.md:297]: nit:
```suggestion
This field applies to all Pods in the range `0` to `replicas - 1`. If there is any
unavailable Pod in the range `0` to `replicas - 1`, it will be counted towards
`maxUnavailable`.
```

The space makes it more clear that we're using code style in place of equation style (which this site doesn't support).

**Review by sftim — APPROVED** [review]
/approve

/hold
I mentioned a nit; feel free to either tweak that or to unhold and ship the changes as-is.

File: content/en/docs/concepts/workloads/controllers/statefulset.md
Comment (tengqm) [reply content/en/docs/concepts/workloads/controllers/statefulset.md:297]: OK

**Comment by tengqm** [comment]
/hold cancel
Nits resolved. Need another lgtm.

**Comment by sftim** [comment]
Thanks @tengqm
/lgtm

Ship it

**Comment by sftim** [comment]
(BTW changes since https://github.com/kubernetes/website/pull/32596#issuecomment-1087641817 are trivial)

**Comment by krmayankk** [comment]
I wanted to write the second blog in this series, do i need to follow a process or i can go ahead and make a PR ?

**Comment by sftim** [comment]
> I wanted to write the second blog in this series, do i need to follow a process or i can go ahead and make a PR ?

This PR didn't add a blog article. Maybe ask your question in #sig-docs-blog on Slack @krmayankk, and mention the URL of the article that you see as being first in a series.

**Comment by sftim** [comment]
PS. Ideally, if you're starting a series of articles, flag that to blog editors when you're looking to publish the _first_ article.

**Pull request merged**
